platforms/shared/configuration/roles/setup/certmanager/tasks/deploy_issuer.yaml

```diff
@@ -0,0 +1,54 @@
+---
+##Creates namespace "{{ components_ns }}"
+- name: create namespace "{{ component_ns }}"
+  kubernetes.core.k8s:
+    kubeconfig: "{{ kubeconfig_path }}"
+    state: present
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: "{{ component_ns }}"
+
+
+## creates Issuer in "{{ component_ns }}" namespace
+- name: create Issuer
+  kubernetes.core.k8s:
+    kubeconfig: "{{ kubeconfig_path }}"
+    state: present
+    definition:
+      apiVersion: cert-manager.io/v1
+      kind: Issuer
+      metadata:
+        name: letsencrypt-prod-{{ component_ns }}
+        namespace: "{{ component_ns }}"
+      spec:
+        acme:
+          email: "{{ email }}"
+          server: https://acme-v02.api.letsencrypt.org/directory
+          privateKeySecretRef:
+            name: issuer-account-key-{{ component_ns }}
+          solvers:
+            - http01:
+                ingress:
+                  class: ambassador
```

Comment:
don't limit this to letsencrypt issuer, expect the issuer to be already present in the cluster.